buildSrc/src/main/java/com/yelp/gradle/bento/GlobalDependencies.kt

```diff
@@ -5,7 +5,7 @@ import java.net.URI
 
 object Publishing {
     const val GROUP = "com.yelp.android"
-    const val VERSION = "15.1.1"
+    const val VERSION = "15.1.2"
```

Comment:
This should be a minor version bump since it adds new functionality. See https://semver.org/ for more info.